src/lib/isISBN.js

```diff
@@ -1,43 +1,55 @@
 import assertString from './util/assertString';
 
-const isbn10Maybe = /^(?:[0-9]{9}X|[0-9]{10})$/;
-const isbn13Maybe = /^(?:[0-9]{13})$/;
+const possibleIsbn10 = /^(?:[0-9]{9}X|[0-9]{10})$/;
+const possibleIsbn13 = /^(?:[0-9]{13})$/;
 const factor = [1, 3];
 
-export default function isISBN(str, version = '') {
-  assertString(str);
-  version = String(version);
-  if (!version) {
-    return isISBN(str, 10) || isISBN(str, 13);
+export default function isISBN(isbn, options) {
+  assertString(isbn);
+
+  // For backwards compatibility:
+  // isISBN(str [, version]), i.e. `options` could be used as argument for the legacy `version`
+  const version = String(options?.version || options);
+
+  if (!(options?.version || options)) {
+    return isISBN(isbn, { version: 10 }) || isISBN(isbn, { version: 13 });
   }
-  const sanitized = str.replace(/[\s-]+/g, '');
+
+  const sanitizedIsbn = isbn.replace(/[\s-]+/g, '');
```

Comment:
a bit off topic for the PR;
I would like to see that "sanitization" step replaced with as option, maybe something like a "strict" mode, checking/ignoring for spaces or hyphens *only inside* the ISBN, not also at the beginning or end of the ISBN string.
but that shouldn't be part of this PR, just noting it down for the future

Reply:
Yes, I had that in mind as well when working on this but from personal experience people write the ISBN down differently. I haven't checked if there are rules on where the - should be, but I think that in this case sanitising whitespaces and - adds value to this specific validator.

Reply:
no, don't worry about it here, IMHO that would be out of scope for this particular PR - let's keep it simple and get this merged and then after the release, we can work on that via a new PR, I'd say